.github/workflows/ci.yml

```diff
@@ -116,6 +116,16 @@ jobs:
       run: ./mill -i integration-core.native-mostly-static.test
     - name: Docker integration tests
       run: ./mill integration.docker.test
+    - name: Login to GitHub Container Registry
```

Comment:
I think these two should have a condition like
```yml
if: github.event_name == 'push'
```
so that they're not run during PRs.